tests/utils/verification/volumeproperties.go

```diff
@@ -132,19 +136,22 @@ func VerifyAttachedStatus(name, hostName, esxName string) bool {
 	return isMatching
 }
 
-// GetVolumeStatusHost - get the volume status on a given host
-func GetVolumeStatusHost(name, hostName string) string {
+// getVolumeStatusHost - get the volume status on a given host
+func getVolumeStatusHost(name, hostName string) string {
 	cmd := dockercli.InspectVolume + " --format '{{index .Status.status}}' " + name
-	op := ExecCmd(hostName, cmd)
-	return strings.TrimSpace(op)
+	// ignoring the error here, helper is part of polling util
+	// error most likely to be "unable to reach host [ssh:255 error]"
+	// VerifyDetachedStatus takes care of retry mechanism
+	out, _ := ssh.InvokeCommand(hostName, cmd)
```

Comment:
so far it is a private member and there is only one caller .. we can enhance if needed in future.